src/main/java/entities/ComputerPlayer.java

```diff
@@ -0,0 +1,38 @@
+package entities;
+
+import java.util.*;
+
+public class ComputerPlayer extends Player {
+
+    String name;
```

Comment:
specify access modifier

Reply:
This has been updated in the previous push.